src/js/apps/forms/form/form-patient_app.js

```diff
@@ -86,7 +86,6 @@ export default App.extend({
     if (!this.isReadOnly) this.bindEvents(formService, this.serviceEvents);
   },
   serviceEvents: {
-    'submit': 'onFormServiceSubmit',
```

Comment:
There's a `this.trigger('submit');` here: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/services/forms.js#L321.

Do we still need that? I don't think so, but I could be mistaken.